src/app/core/site-footer/site-footer/site-footer.component.scss

```diff
@@ -37,12 +37,14 @@ footer {
     grid-column-gap: 20px;
     grid-row-gap: 20px;
     margin-top: 40px;
+    height: 20px;
     @include media-breakpoint-up(lg) {
       display: flex;
       flex-wrap: wrap;
       margin-top: 20px;
     }
     a {
+      cursor: pointer;
```

Comment:
proponuję rozważyć użycie np. [routerLink] bo aktualnie mamy <a> bez "href" i sterowanie kliknięciem do "O projekcie" w handlerze - przez co fixujemy to tutaj CSSem.